server/routes/testutils/appSetup.ts

```diff
@@ -21,9 +21,11 @@ function appSetup(route: Router, production: boolean, hmppsAuthMFAUser: boolean)
     req.user = {
       employeeName: 'first last',
       authSource: 'nomis',
-      token: 'token',
+      token:
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6ImZpcnN0IGxhc3QiLCJpYXQiOjE1MTYyMzkwMjIsImF1dGhvcml0aWVzIjoiUk9MRV9NRkEifQ.-oYWVIbTyzjcrd9XqG5zyh5nf9G_zcLrXy2BxCphexM',
```

Comment:
sample token generated from https://jwt.io/ that just contains `name` and `authorities`

Reply:
There is a security scan of github which might pick this up (even though it is a false positive) . Could the token be generated dynamically?